src/uint256.h

```diff
@@ -91,13 +91,13 @@ class base_blob
                ((uint64_t)ptr[7]) << 56;
     }
 
-    uint32_t GetUint32(int pos) const
+    uint32_t GetUint32LE(int pos) const
```

Comment:
In commit "Implement a signer that takes an unserialized PSBT and signs it":

Was this meant to go into the previous commit?

Reply:
Oops, yes. Fixed